Dockerfile

```diff
@@ -1,6 +1,9 @@
+ARG GIT_POINTER=master
+ARG PYTHON_VERSION=3.6
 
-FROM python:3.6
+FROM python:${PYTHON_VERSION}
+MAINTAINER tyler@prefect.io
 
-RUN pip install prefect
-
-RUN mkdir /root/.prefect/
+RUN apt-get update && apt-get install -y graphviz
+RUN pip install git+https://github.com/PrefectHQ/prefect.git@master#egg=prefect[kubernetes,viz]
```

Comment:
So, I realize that I am the one that suggested this in the first place, but since it doesn't work I vote we remove the `RUN apt-get update && apt-get install -y graphviz` line and the `viz` requirement so this section just looks like:
```
RUN pip install git+https://github.com/PrefectHQ/prefect.git@master#egg=prefect[kubernetes]
```

Reply:
fixed